app/client/components/form/WrappedInput.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-import * as styles from "./styles.module.css";
+import styles from "./styles.module.css";
```

Comment:
fixes type issue, see: https://github.com/TeamSupercell/typings-for-css-modules-loader/issues/8